test/data/scm.addWebhook.yaml

```diff
@@ -2,3 +2,4 @@
 scmUri: github.com:126987453:master
 token: thisisashinytoken
 webhookUrl: https://screwdriver.cd/v4/some-endpoint
+scmContext: github.com
```

Comment:
These scmContexts do not look valid compared to the ones in the other PR. Please be consistent.